src/azul/openapi/schema.py

```diff
@@ -220,3 +225,27 @@ def make_type(t: TYPE) -> JSON:
         return t
     else:
         assert False, type(t)
+
+
+def enum(*items: PrimitiveJSON, type_: TYPE = None) -> JSON:
+    """
+    Returns a JSON schema `enum` items of a given type.
+
+    >>> from azul.doctests import assert_json
+    >>> assert_json(enum('foo', 'bar', type_=str))
+    {
+        "type": "string",
+        "enum": [
+            "foo",
+            "bar"
+        ]
+    }
+    """
+    if type_:
```

Comment:
`is None` or `is not None`. This keeps popping up in your contributions, please try to remember the distinction between `bool(x)` and `x is None`. It may not matter here because no instance of `type` should ever be False-ish but it's just not good practice to rely on the complicated assumptions. If you want to test whether something is `None` use `is None`.

Reply:
According to the function signature, `type_` can be a string or a dictionary. This assertion wouldn't work correctly in those cases. The assertion should only be made if `type_` is an instance of `type`, otherwise we should trust the caller.